helm/oaf/templates/MTN-RW-connector.yaml

```diff
@@ -0,0 +1,90 @@
+{{- if .Values.mtn_rwanda_connector.enabled -}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "mtn-rw-connector"
+  labels:
+    org.oneacrefund.project: mtn-rwanda-connector
+    org.oneacrefund.app: {{ .Release.Name }}-connector
+    org.oneacrefund.instance: {{ .Release.Name }}-connector    
+spec:
+  replicas: {{ .Values.mtn_rwanda_connector.replicas }}
+  selector:
+    matchLabels:
+      org.oneacrefund.project: mtn-rwanda-connector
+      org.oneacrefund.app: {{ .Release.Name }}-connector
+      org.oneacrefund.instance: {{ .Release.Name }}-connector
+  template:
+    metadata:
+      labels:
+        app: mtn-rw-connector
+      annotations:
```

Comment:
Add provision to add more annotations by overbidding values. The env_template repo does this very well.